bench/index.js

```diff
@@ -0,0 +1,101 @@
+global.fs = require('fs');
```

Comment:
The use of benchmark for stylus-loader is kind of weird. `benchmark` handles all the funny math around benchmarking nicely but is geared towards benchmarking small tid bits of javascript. To better work with those small tid bits and to see js performance like it was the first pass and not highly JITed performance (unless that happens during sampling or something, idk its weird and I trust the idea but its not useful here) `benchmark` evals what you are testing.

For some reason webpack was available for the first benchmark for the second didn't want to recognize fs. So I stored it on the global. I'll look a little further tonight. I'd also be happen to pop these benchmark commits out into another branch.